charts/graylog/values.yaml

```diff
@@ -55,11 +55,12 @@ graylog:
   ##
   image:
     repository: "graylog/graylog"
-    tag: "4.2.3-1"
+    tag: "5.0.2"
     pullPolicy: "IfNotPresent"
 
   ## Graylog default Java option
-  javaOpts: "-Dlog4j2.formatMsgNoLookups=true -Djdk.tls.acknowledgeCloseNotify=true -XX:+UnlockExperimentalVMOptions -XX:NewRatio=1 -XX:MaxMetaspaceSize=256m -server -XX:+ResizeTLAB -XX:-OmitStackTraceInFastThrow"
+  # javaOpts: "-Dlog4j2.formatMsgNoLookups=true -Djdk.tls.acknowledgeCloseNotify=true -XX:+UnlockExperimentalVMOptions -XX:-OmitStackTraceInFastThrow -XX:+UseG1GC -server"
+  javaOpts: "-Dlog4j2.formatMsgNoLookups=true -Djdk.tls.acknowledgeCloseNotify=true -XX:+UnlockExperimentalVMOptions -XX:-OmitStackTraceInFastThrow -XX:+UseG1GC -server"
```

Comment:
I had problems with the following parameter:
-XX:+UseCGroupMemoryLimitForHeap
It is being set on line [109 charts/graylog/templates/statefulset.yaml](https://github.com/KongZ/charts/blob/main/charts/graylog/templates/statefulset.yaml#L109)
After removing it worked for me.
Be sure it works.
More info in the graylog repo:
[Link to Graylog Repo](https://github.com/Graylog2/graylog-docker/blob/5.0/docker/oss/Dockerfile#L67)